src/spec.c

```diff
@@ -141,8 +152,10 @@ int IndexSpec_CheckAllowSlopAndInorder(const IndexSpec *spec, t_fieldMask fm, Qu
     if (fm & ((t_fieldMask)1 << ii)) {
       const FieldSpec *fs = spec->fields + ii;
       if (FIELD_IS(fs, INDEXFLD_T_FULLTEXT) && (FieldSpec_IsUndefinedOrder(fs))) {
+        char fieldName[MAX_OBFUSCATED_FIELD_NAME];
+        Obfuscate_Field(fs->index, fieldName);
```

Comment:
Function/macro

Reply:
Are you saying you want me to wrap the array declaration and function call in a macro?